gems/sorbet-runtime/test/types/props/_props.rb

```diff
@@ -172,7 +172,29 @@ class TestRedactedProps
     prop :secret, String, redaction: [:truncate, 4], sensitivity: []
   end
 
+  describe 'redacted props with no redaction handler' do
+    it 'raises when fetching redacted values' do
+      assert_raises do
+        d = TestRedactedProps.new
+        d.str = '12345'
+
+        # this will raise an error without a redaction handler
+        d.str_redacted
+      end
+    end
+  end
+
   describe 'redacted props' do
+    before do
+      T::Configuration.redaction_handler = lambda do |value, redaction|
+        Chalk::Tools::RedactionUtils.redact_with_directive(value, redaction)
```

Comment:
Optional: Might be good to decouple this test from `Chalk::Tools::RedactionUtils` by defining a redaction function locally.

Reply:
This version is defined locally in `test_helper.rb`